apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```diff
@@ -3,20 +3,29 @@ import PropTypes from 'prop-types';
 import {curriculumDataShape} from './curriculumCatalogShapes';
 import i18n from '@cdo/locale';
 import style from '../../../style/code-studio/curriculum_catalog_container.module.scss';
-import HeaderBanner from '../HeaderBanner';
 import CourseCatalogIllustration01 from '../../../static/curriculum_catalog/course-catalog-illustration-01.png';
 import CurriculumCatalogCard from '@cdo/apps/templates/curriculumCatalog/CurriculumCatalogCard';
 import {translatedCourseOfferingDurations} from '@cdo/apps/templates/teacherDashboard/CourseOfferingHelpers';
 
 const CurriculumCatalog = ({curriculaData}) => (
   <>
     <div className={style.catalogHeader}>
-      <HeaderBanner
-        headingText={i18n.curriculumCatalogHeaderTitle()}
-        subHeadingText={i18n.curriculumCatalogHeaderSubtitle()}
-        short={false}
-        imageUrl={CourseCatalogIllustration01}
-      />
+      <div className={style.catalogHeaderContent}>
+        <div className={style.headerTextWrapper}>
+          <h1 className={style.headerTitle}>
+            {i18n.curriculumCatalogHeaderTitle()}
+          </h1>
+          <p className={style.headerSubtitle}>
+            {i18n.curriculumCatalogHeaderSubtitle()}
+          </p>
+        </div>
+        <div className={style.headerImageContainer}>
+          <img
+            className={style.headerImage}
+            src={CourseCatalogIllustration01}
```

Comment:
Can you add an empty alt text so that screenreaders don't read this?

Reply:
Yes thank you for catching that!